docs/guides/use_custom_images.md

```diff
@@ -0,0 +1,23 @@
+To use custom images when building a deployment you will have to use a certain environment variables. There are total 5 docker images used in a deployment setup.
```

Comment:
First sentence doesn't make sense grammatically. Not 100% sure what you are trying to say.